rep-2000.rst

```diff
@@ -1179,6 +1179,120 @@ Package manager use for dependencies:
 - OpenEmbedded: opkg
 
 
+Build System Support:
+
+- ament_cmake
+- cmake
+- setuptools
+
+Kilted Kaiju (May 2025 - November 2026)
+---------------------------------------
+
+Targeted platforms:
+
++--------------+------------------+---------------+------------------+-------------+-----------------+----------------+
+| Architecture | Ubuntu Noble     | Windows 10    | RHEL 9           |  macOS      | Debian Bookworm | OpenEmbedded / |
+|              | (24.04)          | (VS2019)      |                  |             | (12)            | Yocto Project  |
```

Comment:
Are we planning to bump to VS2022?
```suggestion
|              | (24.04)          | (VS2022)      |                  |             | (12)            | Yocto Project  |
```

Reply:
I'm hoping that is the case, but we haven't gotten it to work yet.  So let's leave this as 2019 for now, and we can update it later if we actually get that work done.